src/components/Home/ProfileSelector/ProfileSelector.spec.ts

```diff
@@ -1,6 +1,8 @@
 export interface SelectButtonSpec {
     name?: string,
-    onClick?: React.MouseEventHandler<HTMLButtonElement>,
+    setValue: any,
+    value: any,
+    list: any,
```

Comment:
We're not gonna use "any" here, this might lead to some undesired behaviour.

You might want to read that: https://www.tutorialsteacher.com/typescript/typescript-generic-interface

Reach to me if you're having a hard time with this.